src/services/item/plugins/published/plugins/search/meilisearch.ts

```diff
@@ -163,8 +163,9 @@ export class MeiliSearchWrapper {
       content: await this.getContent(item),
       isPublishedRoot: isPublishedRoot,
       isHidden: isHidden,
-      createdAt: item.createdAt,
-      updatedAt: item.updatedAt,
+      // todo: fix these types
```

Comment:
Still need to be fixed?

Reply:
Yes it uses the type from sdk and it expects a string and we give it a Date ...